frontends/web/src/locales/en/app.json

```diff
@@ -34,7 +34,8 @@
     "verify": "Verify on device",
     "xpubTypeChangeBtn": {
       "p2pkh": "View legacy P2PKH extended public key",
-      "p2wpkh": "View Native segwit",
+      "p2tr": "View taproot",
+      "p2wpkh": "View native segwit",
```

Comment:
i changed this on locize already

Reply:
LGTM, but I'd always use capital `T` for the name `Taproot`

Reply:
:+1: done